app/main.py

```diff
@@ -1,6 +1,27 @@
+import os
+import requests
+
+
+URL = "http://api.weatherapi.com/v1/current.json/?"
+API_KEY = os.environ.get("API_KEY")
+CITY = "Paris"
+parameters = {"key": API_KEY, "q": CITY}
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    try:
```

Comment:
if you want to use try block, remember that you should write only code that can cause errors inside it.

Reply:
Here I decided to use try/except (for sure) because I use dict[key] not dict.get(key).